### DISCUSSION TITLE
如何针对一个特定的网站应用一个服务节点？其它网站走另外的服务节点？

### DISCUSSION THREAD

**Discussion by xianghongai** [v2rayA/v2rayA#864] [Q&A]
如：
- https://chat.openai.com ，走美区节点；
- https://www.google.com ，这类站点走香日韩新台节点。

**Comment by mzz2017** [comment]
v2raya.org 里面有，看高级应用

**Comment by zuilintan** [comment]
https://v2raya.org/docs/advanced-application/multiple-outbound-routing/
